Assets/MRTK/SDK/Features/UX/Scripts/Loader/LoaderController.cs

```diff
@@ -196,10 +196,8 @@ private float Cycles
                 {
                     return (frequency < 0.0f) ? (Time.time / 0.000001f) * -1.0f : (Time.time / frequency) * -1.0f;
                 }
-                else
-                {
-                    return (frequency < 0.0f) ? Time.time / 0.000001f : Time.time / frequency;
-                }
+                
+                return (frequency < 0.0f) ? Time.time / 0.000001f : Time.time / frequency;
```

Comment:
I think this should get reverted back to the older pattern, since they should be equivalent functionally and the `else` helps make the code clearer.

Reply:
Okay understood